packcheck.sh

```diff
@@ -1206,7 +1151,7 @@ ensure_cabal_config() {
     run_verbose cabal user-config init || cabal info . > /dev/null || true
```

Comment:
1. We can now check cabal version and die if it is older than the one where we have v2 commands.
2. We can remove the `cabal info` command above.